src/taxes/pages/TaxCountriesPage/TaxCountriesMenu/styles.ts

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from "@saleor/macaw-ui";
+
+export const useStyles = makeStyles(
+  theme => ({
+    clickable: {
+      cursor: "pointer"
+    },
+    scrollWrapper: {
+      overflow: "scroll",
```

Comment:
Both x and y?

Reply:
+1 we don't have that on channels list
![CleanShot 2022-06-03 at 10 19 47@2x](https://user-images.githubusercontent.com/4144459/171816845-356dc896-3c53-497f-82fd-f92621c987c8.png)